src/init.cpp

```diff
@@ -438,6 +439,15 @@ void SetupServerArgs()
     gArgs.AddArg("-torcontrol=<ip>:<port>", strprintf("Tor control port to use if onion listening enabled (default: %s)", DEFAULT_TOR_CONTROL), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     gArgs.AddArg("-torpassword=<pass>", "Tor control port password (default: empty)", ArgsManager::ALLOW_ANY | ArgsManager::SENSITIVE, OptionsCategory::CONNECTION);
     gArgs.AddArg("-asmap=<file>", "Specify asn mapping used for bucketing of the peers. Path should be relative to the -datadir path.", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+#ifdef USE_NATPMP
+#if USE_NATPMP
+    gArgs.AddArg("-natpmp", "Use NAT-PMP to map the listening port (default: 1 when listening and no -proxy)", ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
+#else
+    gArgs.AddArg("-natpmp", strprintf("Use NAT-PMP to map the listening port (default: %u)", 0), ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
```

Comment:
`%s` > `%u`

Reply:
> `%s` > `%u`

What is the rationale? We already use the `%u` specifier for unsigned integers in `tfm::format()`.

Reply:
`%s` doesn't care what type the value is.

You can use the same format string above, with a value of `"1 when listening and no -proxy"`

Reply:
Done in the latest push.